benchmarks/src/jvmMain/kotlin/pro/respawn/flowmvi/benchmarks/BenchmarkDefaults.kt

```diff
@@ -0,0 +1,6 @@
+package pro.respawn.flowmvi.benchmarks
+
+internal object BenchmarkDefaults {
+
+    const val intentsPerIteration = 1000
```

Comment:
I'm using 5_000 in Fluxo benchmarks :)

Reply:
Got it. I'll keep a 1000 for easier measurement of throughput